www/source/partials/docs/_glossary-supervisor.html.md.erb

```diff
@@ -18,3 +18,5 @@ An example is leader election: it's handled here by having a consistent (and sim
 ## Supervisor REST API
 
 The Habitat Supervisor provides a HTTP API to expose cluster metadata, statistics, and general diagnostic information useful for monitoring and support in the form of a JSON document. It also provides detailed information about the Habitat package that it is supervising, including metadata such as the build and runtime dependencies and their versions.
+
+<%= partial '/partials/docs/using-hab-monitor-services' -%>
```

Comment:
I'm not certain that we want this here. I can be convinced either way but a majority of the glossary content is glorified definitions as opposed to functional documentation.

Reply:
It doesn't make sense to refer to an API and give no information about the endpoints or where to look them up. This is problematic across the docs right now. We mention components with no information about what they are. I played with the idea of simply linking to the glossary, but thinking through what the user interaction for a new user would be, you would end up with six tabs open to have a coherent learning experience, rather than simply being able to scroll through one page that's guiding you through the different concepts.